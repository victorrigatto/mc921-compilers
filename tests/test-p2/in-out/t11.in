int main(int i, char c) {
  int i, j
  j = i + 1;
}
