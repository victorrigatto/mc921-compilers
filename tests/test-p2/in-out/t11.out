ParserError: Before j @ 3:3
